lib/game.rb

```diff
@@ -15,4 +15,8 @@ def play
       |__|__|__|
     GRID
   end
+
+  private
```

Comment:
You know I just had a conversation with Derrick about this in a code review. Private doesn't exist in the small talk world were a lot of great OO concepts came from. And so private methods for me are a line I draw to let people know, they shouldn't worry about these, but also that maybe they don't belong in this class. Making methods private via extracting to a helper class is what I was thinking. I do use the `private` indicator, but not in this way.

It's been a very weird thing to see in thoughtbot the practice of making private accessors and it hearkens back to Java for me. I know it works for people though. So wondering how it help you, genuinely and without value judgement?

By the way this is really fun to talk about. So thanks for getting me to review.

Reply:
You know, I think I've been parroting this since my early days at thoughtbot. This matches up with how I'm viewing it:

> a line I draw to let people know, they shouldn't worry about these

It also feels like revealing the intentions, while sort of issuing a warning of "I'm not responsible if you cross this line."

I'm curious to hear more about how you'd use a helper class here!

Reply:
Something that's been on my todo list for many years now is to learn more about Smalltalk. Any suggestions for resources there?

Reply:
I don't have resources for Small talk :( I haven't learned it yet. So, I win the consulting interview for talking tradeoffs that I haven't actually experienced.

I guess with the private demarkation, I think of it as a junk drawer of methods waiting to be sorted into real responsibilities. So, when I start to see them coalesce across objects or within one, I extract it out into a single responsibility.